src/trading-engine/orderbook.event.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
   },
 
   createNewReasonObject: (originalOrderEvent) => {
-    const orderEvent = JSON.parse(JSON.stringify(originalOrderEvent));
+    const orderEvent = JSON.parse(JSON.stringify(originalOrderEvent)); // [Tung]: why unnecessary serializing and de-serializing of objects? All event objects in orderbook.js are already parsed to real OrderEvent-objects
```

Comment:
why unnecessary serializing and de-serializing of objects? All event objects in orderbook.js are already parsed to real OrderEvent-objects?